src/main/java/org/ld4l/bib2lod/managers/SimpleManager.java

```diff
@@ -40,21 +44,32 @@ public static void main(String[] args) {
         LOGGER.info("START CONVERSION.");
 
         try {
-            setup(args);
-            convert();
+        	SimpleManager fm = new SimpleManager(args);
+        	fm.convert();
             LOGGER.info("END CONVERSION.");
         } catch (Exception e) {
             LOGGER.error(e.getMessage());
             e.printStackTrace();
             LOGGER.error("CONVERSION FAILED TO COMPLETE");
         }
     }
-
+    
+    /*
+     * Private constructor - do not use since configuration will not have been set up.
+     */
+    @SuppressWarnings("unused")
+	private SimpleManager() {
+    	super();
+    }
+    
     /**
      * Parse the command line options, read the config file and adjust as
-     * necessary. Set up the object factory.
+     * necessary.
+     * 
+     * @param args - Command line arguments most likely passed from main().
      */
-    private static void setup(String[] args) {
+    public SimpleManager(String[] args) {
+    	super();
```

Comment:
I don't know what purpose is served by invoking super() in a class that extends Object. Stylistic preference?